dll/win32/syssetup/syssetup.rc

```diff
@@ -33,7 +33,23 @@ IDI_ICON5 ICON "resources/5.ico"
 
 STRINGTABLE
 BEGIN
-    IDS_ACKPROJECTS "Wine - http://www.winehq.org\nFreeType - http://www.freetype.org\nSYSLINUX - http://syslinux.zytor.com\nMinGW - http://www.mingw.org\nBochs - http://bochs.sourceforge.net\nMesa3D - http://www.mesa3d.org\nadns - http://www.gnu.org/software/adns\nICU - http://www.icu-project.org/\nGraphApp - http://enchantia.com/software/graphapp/\nExt2 - http://www.ext2fsd.com/\nGNU FreeFont - http://savannah.gnu.org/projects/freefont/\nDejaVu Fonts - http://dejavu.sourceforge.net\nLiberation(tm) Fonts - https://fedorahosted.org/liberation-fonts/\nBtrfs - https://github.com/maharmstone/btrfs\nTango Desktop Project - http://tango.freedesktop.org/Tango_Desktop_Project"
+    IDS_ACKPROJECTS "\
+Wine - http://www.winehq.org\
+\nFreeType - http://www.freetype.org\
+\nSYSLINUX - http://syslinux.zytor.com\
+\nMinGW - http://www.mingw.org\
+\nBochs - http://bochs.sourceforge.net\
+\nMesa3D - http://www.mesa3d.org\
+\nadns - http://www.gnu.org/software/adns\
+\nICU - http://www.icu-project.org\
+\nGraphApp - http://enchantia.com/software/graphapp\
+\nExt2 - http://www.ext2fsd.com\
+\nGNU FreeFont - http://savannah.gnu.org/projects/freefont\
+\nDejaVu Fonts - http://dejavu.sourceforge.net\
+\nLiberation(tm) Fonts - https://fedorahosted.org/liberation-fonts\
+\nBtrfs - https://github.com/maharmstone/btrfs\
+\nTango Desktop Project - http://tango.freedesktop.org/Tango_Desktop_Project\
```

Comment:
It would actually be of interest to have a trailing newline in text when this is displayed in the scrollable box;
in which case that line could have a terminating `\n`, and you could reformat all the others by moving their `\n` at the end as well.
I'll let you test that suggestion to see whether it displays "better" or not.

Reply:
No I intentionally did not do that. I deliberately formatted like that with the goal to **not** have the \n at line end. Only that way a new line can be added without touching the previous line (and therefore destroying the git blame information unnecessarily).